web/src/app/core/utils/custom-cron-parser.ts

```diff
@@ -0,0 +1,68 @@
+import {CronToDatetimeConverter} from "../../datetime-cron-converter/cron-to-datetime.converter";
+
+export enum CronConvertStrategy {DEFAULT, PERIOD_SENSITIVE}
+
+export interface CronParseResult {
+    hours: string;
+    minutes: string;
+    period: string;
+    time: string;
+}
+
+const HOURS_MAP = {24: 12, 23: 11, 22: 10, 21: 9, 20: 8, 19: 7, 18: 6, 17: 5, 16: 4, 15: 3, 14: 2, 13: 1};
```

Comment:
Looks like it could be substitute with some formula
Smth like `24 - / (hours mod 12)`